client-spark/spark-3/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -99,6 +99,12 @@ public SparkShuffleManager(SparkConf conf, boolean isDriver) {
               + "use Celeborn as Remote Shuffle Service to avoid performance degradation.",
           SQLConf.LOCAL_SHUFFLE_READER_ENABLED().key());
     }
+    if (conf.getBoolean("spark.dynamicAllocation.shuffleTracking.enabled", false)) {
```

Comment:
[SPARK-39846](https://issues.apache.org/jira/browse/SPARK-39846) changed the default value from false to true

Reply:
Should we use the below 
```java
    if ((Boolean) conf.get(package$.MODULE$.DYN_ALLOCATION_SHUFFLE_TRACKING_ENABLED())) {
      logger.warn(
          "Detected {} (default is {}) is enabled, it's highly recommended to disable it when "
              + "use Celeborn as Remote Shuffle Service to avoid performance degradation.",
          package$.MODULE$.DYN_ALLOCATION_SHUFFLE_TRACKING_ENABLED().key(),
          package$
              .MODULE$
              .DYN_ALLOCATION_SHUFFLE_TRACKING_ENABLED()
              .defaultValue()
              .map(v -> (Boolean) v));
    }
```